<launch>
<param name="video_source" value="$(find windMill)/../../../../samples/armor/red-p.mov"/>
    <node pkg="windMill" type="windMill_node" name="windMill" output="screen" >
        <remap from="/MVCamera/image_raw" to ="/camera/image_raw"/>
     </node>
    <node pkg="video_pub" type="video_pub_node" name="video_pub" output="screen" respawn="true" />
<!--    <node pkg="numPred" type="numPred_node.py" name="numPred" output="screen" />-->



<node pkg="serial_common" type="serial_common_node" name="serial_common_node" output="screen" />

<rosparam command="load" file="$(find windMill)/cfg/params.yaml" />

</launch>
